src/vue/modules/create-asset-sale/components/create-asset-sale-form.vue

```diff
@@ -0,0 +1,737 @@
+            </tick-field>
+          </div>
+        </div>
+        <div class="create-asset-sale-form__error-text">
+          {{ getFieldErrorMessage('form.saleInformation.quoteAssets') }}
+        </div>
+        <div class="app__form-actions">
+          <button
+            v-ripple
+            type="submit"
+            class="app__button-raised"
+            :disabled="formMixin.isDisabled"
+          >
+            {{ 'create-asset-sale-form.next-btn' | globalize }}
+          </button>
+        </div>
+      </form>
+      <form
+        novalidate
+        v-if="currentStep === STEPS.shortBlurb.number"
+        class="app__form"
+        @submit.prevent="isFormValid() && showConfirmation()"
+      >
+        <div class="app__form-row">
+          <div class="app__form-field">
+            <file-field
+              :label="'create-asset-sale-form.cover-logo' | globalize"
+              :note="'create-asset-sale-form.upload-image' | globalize"
+              name="create-sale-sale-logo"
+              accept=".jpg, .png"
+              :document-type="DOCUMENT_TYPES.saleLogo"
+              v-model="form.shortBlurb.saleLogo"
+              :disabled="formMixin.isDisabled"
+              :error-message="getFieldErrorMessage(
+                'form.shortBlurb.saleLogo'
+              )"
+            />
+          </div>
+        </div>
+        <div class="app__form-row">
+          <div class="app__form-field">
+            {{ 'create-asset-sale-form.short-description' | globalize }}
+            <textarea-field
+              id="sale-short-description"
+              name="create-sale-short-description"
+              v-model="form.shortBlurb.shortDescription"
+              @blur="touchField('form.shortBlurb.shortDescription')"
+              :label="'create-asset-sale-form.subject-lbl' | globalize({
+                length: DESCRIPTION_MAX_LENGTH
+              })"
+              :maxlength="DESCRIPTION_MAX_LENGTH"
+              :error-message="getFieldErrorMessage(
+                'form.shortBlurb.shortDescription', {
+                  length: DESCRIPTION_MAX_LENGTH
+                }
+              )"
+            />
+          </div>
+        </div>
+        <div class="app__form-row">
+          <div class="app__form-field">
+            {{ 'create-asset-sale-form.full-description' | globalize }}
+            <description-editor
+              v-model="form.shortBlurb.description"
+            />
+          </div>
+        </div>
+        <div class="app__form-actions">
+          <button
+            v-ripple
+            v-if="!formMixin.isConfirmationShown"
+            type="submit"
+            class="app__button-raised"
+            :disabled="formMixin.isDisabled"
+          >
+            {{ 'create-asset-sale-form.submit-btn' | globalize }}
+          </button>
+          <form-confirmation
+            v-if="formMixin.isConfirmationShown"
+            @ok="hideConfirmation() || submit()"
```

Comment:
Use only `@ok="submit"` here. Also, create a variable that will look like `isFormSubmitting`, set it in `true/false` in the `submit` method and pass it here via `isPending` prop. `FormConfirmation` component will use this prop and make some things inside. Don't forget to use `hideConfirmation()` at the end of the `submit` method.

Reply:
ok